doc/rvv-intrinsic-spec.adoc

```diff
@@ -0,0 +1,523 @@
+* `_tum` suffix: Represents a masked (`vm=0`) vector operation with tail-undisturbed (`vta=0`), mask-agnostic (`vma=1`)
+* `_mu` suffix: Represents a masked (`vm=0`) vector operation with tail-agnostic (`vta=1`), mask-undisturbed (`vma=0`)
+* `_tumu` suffix: Represents a masked (`vm=0`) vector operation with tail-undisturbed (`vta=0`), mask-undisturbed (`vma=0`)
+
+Using `vadd` with SEW=8 and LMUL=1 as an example, the variants are:
+
+```c
+// vm=1, vta=1
+vint32m1_t __riscv_vadd_vv_i32m1(vint32m1_t op1, vint32m1_t op2, size_t vl);
+// vm=1, vta=0
+vint32m1_t __riscv_vadd_vv_i32m1_tu(vint32m1_t maskedoff, vint32m1_t op1,
+                                    vint32m1_t op2, size_t vl);
+// vm=0, vta=1, vma=1
+vint32m1_t __riscv_vadd_vv_i32m1_m(vbool32_t mask, vint32m1_t op1,
+                                   vint32m1_t op2, size_t vl);
+// vm=0, vta=0, vma=1
+vint32m1_t __riscv_vadd_vv_i32m1_tum(vbool32_t mask, vint32m1_t maskedoff,
+                                     vint32m1_t op1, vint32m1_t op2, size_t vl);
+// vm=0, vta=1, vma=0
+vint32m1_t __riscv_vadd_vv_i32m1_mu(vbool32_t mask, vint32m1_t maskedoff,
+                                    vint32m1_t op1, vint32m1_t op2, size_t vl);
+// vm=0, vta=1, vma=1
+vint32m1_t __riscv_vadd_vv_i32m1_tumu(vbool32_t mask, vint32m1_t maskedoff,
+                                      vint32m1_t op1, vint32m1_t op2,
+                                      size_t vl);
+```
+
+NOTE: When policy is set to "agnostic", users may either get tail (inactive maskedoff) element of undisturbed or all ones. Under this policy users should not assume the values within to be deterministic.
+
+NOTE: Pseudo intrinsics mentioned under <<chapter-pseudo-instruction>> do not map to real vector intsructions. Therefore these intrinsics are not affected by the policy setting, nor do they have intrinsic variants of the policy suffix listed above.
+
+=== Passthrough operands in the intrinsics
+
+Intrinsics whose computation is relevant to value held in `vd` have a passthrough operand in them. The following list enumerates the intrinsics that has a passthrough operand. Please see (Appendix: list of prototypes of intrinsics) for the exact prototypes.
+
+- Intrinsics with tail-undisturbed (`vta=0`)
+- Intrinsics with mask-undisturbed (`vma=0`)
+- Intrinsics representing Vector Reduction Operations ^13^
+- Intrinsics representing Vector Multiply-Add Operations ^14^
+
+[[explicit-naming-scheme]]
+=== Explicit (Non-overloaded) naming scheme
+
+In general, the intrinsics are encoded as the following. The intrinsics under this naming scheme are the "non-overloaded intrinsics", which in parallel we have the "overloaded intrinsics" defined under <<implicit-naming-scheme>>.
+
+```
+__riscv_{V_INSTRUCTION_MNEMONICS}_{OPERAND_MNEMONICS}_{RETURN_TYPE}_{ROUND_MODE}_{POLICY}{(...)
+```
+
+* `OPERAND_MNEMONICS` are like `vv`, `vx`, `vs`, `vvm`, `vxm`.
+* `RETURN_TYPE` can be split to whether the destination is a mask register
```

Comment:
I don't know what "can be split to whether the destination is a mask register" means here. What does it mean to split in this context?